backend/handler/database/roms_handler.py

```diff
@@ -445,11 +445,10 @@ def filter_roms(
         if group_by_meta_id:
 
             def build_func(provider: str, column: InstrumentedAttribute):
-                if platform_id:
-                    return func.concat(provider, "-", Rom.platform_id, "-", column)
-
                 return func.concat(provider, "-", Rom.platform_id, "-", column)
```

Comment:
For platform variants/versions, those platforms have different `id` values, right? If that's the case, and we want to group them nevertheless, would it be better to remove `Rom.platform_id` from the concat.

Reply:
That would also group games from different platforms with the same metadata IDs, but thinking about it I think that's an even better UX!